### PR TITLE
Ea/coc performance census

### PR DESCRIPTION
**Pull request by eanders**
[//]: # 'remove this if PR is for a release-* branch'
# _Please squash merge this PR_

## Description

Adds census to charts on the CoC Performance Measurement report as appropriate.

## Type of change
[//]: # 'remove options that are not relevant'

- [x] New feature (adds functionality)

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: app/models/grda_warehouse/us_census_api/aggregates.rb [hunk 1]
```diff
@@ -86,39 +84,73 @@ def self.compose(a, b)
...
+      RACE_ETH.each do |stratum|
+        ['MALE', 'FEMALE'].each do |sex|
+          const_set("#{stratum}_#{sex}", ["POP::#{stratum}::#{sex}"])
+        end
+      end
 
       RACE_ETH.each do |stratum|
-        ["MALE", "FEMALE"].each do |sex|
+        ['MALE', 'FEMALE'].each do |sex|
           const_set("#{stratum}_#{sex}_AGE_0_4", ["POP::#{stratum}::#{sex}::UNDER_5_YEARS"])
```
Comment (awisema) [inline app/models/grda_warehouse/us_census_api/aggregates.rb:112]: Maybe make a hash of `AGE_0_4: :UNDER_5_YEARS` etc. and iterate like you did for `RACE_ETH`?

File: drivers/performance_measurement/app/models/performance_measurement/equity_analysis/data.rb [hunk 2]
```diff
@@ -184,10 +195,154 @@ def chart_height
...
+      when 'AmIndAKNative' then 'NATIVE_AMERICAN'
+      when 'Asian' then 'ASIAN'
+      when 'BlackAfAmerican' then 'BLACK'
+      when 'NativeHIPacific' then 'PACIFIC_ISLANDER'
+      when 'White' then 'WHITE'
+      when 'RaceNone' then 'OTHER_RACE'
+      when 'MultiRacial' then 'TWO_OR_MORE_RACES'
+      when 'HispanicLatinaeo' then 'HISPANIC'
+      when 'All' then 'ALL_PEOPLE'
+      end
```
Comment (awisema) [inline drivers/performance_measurement/app/models/performance_measurement/equity_analysis/data.rb:232]: Will `MidEastNAfrican` never turn up? Or is it OK, that it falls through to `nil`?

File: drivers/performance_measurement/app/models/performance_measurement/equity_analysis/data.rb [hunk 3]
```diff
@@ -184,10 +195,154 @@ def chart_height
...
+      when :gender_man then 'MALE'
+      when :gender_woman then 'FEMALE'
+      end
+    end
+
+    private def age_census_code(code)
+      case code
+      when :zero_to_four
+        'AGE_0_4'
+      when :five_to_ten
```
Comment (awisema) [inline drivers/performance_measurement/app/models/performance_measurement/equity_analysis/data.rb]:
```suggestion
      when :five_to_nine
```

File: drivers/performance_measurement/app/views/performance_measurement/warehouse_reports/reports/_equity_analysis.haml [hunk 4]
```diff
@@ -59,3 +59,7 @@
             No report data has been selected to display.
           %span
             Make required selections above and click “update report”
+  - if @analysis_builder.valid?
```
Comment (awisema) [inline drivers/performance_measurement/app/views/performance_measurement/warehouse_reports/reports/_equity_analysis.haml:62]: Should this be
```suggestion
  - unless @analysis_builder.valid?
```
?

**Review by awisema — CHANGES_REQUESTED** [review]

File: app/models/grda_warehouse/us_census_api/aggregates.rb (hunk 1 above)
Comment (eanders) [reply app/models/grda_warehouse/us_census_api/aggregates.rb:112]: I'm going to dry up the sexes, but leave the ages for now, this is a direct copy from some other code, and the constants aren't as straight forward as one might think.

File: drivers/performance_measurement/app/models/performance_measurement/equity_analysis/data.rb (hunk 2 above)
Comment (eanders) [reply drivers/performance_measurement/app/models/performance_measurement/equity_analysis/data.rb:232]: Correct, non-census races will just be zero.  At least for now, that's all we can do.

File: drivers/performance_measurement/app/views/performance_measurement/warehouse_reports/reports/_equity_analysis.haml (hunk 4 above)
Comment (eanders) [reply drivers/performance_measurement/app/views/performance_measurement/warehouse_reports/reports/_equity_analysis.haml:62]: No, that's basically show if we've made some selections.  It's not a warning about poor choices, but a warning about which data is available.

**Review by awisema — APPROVED** [review]

**Pull request merged**